### ISSUE TITLE
Adding .dockerignore file

### ISSUE THREAD

**Issue by samikeijonen** [Seravo/wordpress#212]
I haven't studied yet much about `.dockerignore` file. But what we do know is that Apple M1 Pros are super slow when starting Docker.

Issue could be for example lot's of files in `node_modules` folder. Maybe Adding `.dockerignore` file solves this issue.

**Comment by pekkakortelainen** [comment]
Yes, `node_modules` can get quite bulky. We'll investigate the possibilities `.dockerignore` can offer.

Thanks for opening this issue!
